src/context/ClaimBankFlowContext.tsx

```
@@ -3,9 +3,8 @@
 import React, { createContext, type ReactNode, useContext, useMemo, useState, useCallback } from 'react'
 import { type CountryData } from '../components/AddMoney/consts'
 import { type TCreateOfframpResponse } from '@/services/services.types'
-import { type Account, type User } from '@/interfaces'
+import { type Account, type CounterpartyUser } from '@/interfaces'
```

ESLint

[error] '`@/interfaces`' import is restricted from being used. Import from a specific file instead of the '`@/interfaces`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)